### PR TITLE
Add network related functionality

### PR DESCRIPTION
**Pull request by yuval-fireblocks**
## Pull Request Description

Added flexible network options

Fixes (link to the issue here)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Locally tested against Fireblocks API

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have added corresponding labels to the PR

### CODE REVIEW

File: src/fireblocks-sdk.ts [hunk 1]
```diff
@@ -198,13 +199,96 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest("/v1/network_connections");
     }
 
+    /**
+     * Creates a network connection
+     * @param localNetworkId NetworkId for the local
+     * @param remoteNetworkId NetworkId for the remote
+     * @param routingPolicy of the connection
+     */
+    public async createNetworkConnection(localNetworkId: string, remoteNetworkId: string, routingPolicy?: RoutingPolicy) {
```
Comment (Yinon-Hever) [inline src/fireblocks-sdk.ts:208]: can u add return value pls?

File: src/fireblocks-sdk.ts [hunk 2]
```diff
@@ -198,13 +199,96 @@ export class FireblocksSDK {
...
+     * Deletes a network connection
+     * @param connectionId the connectionId between the peers
+     */
+    public async removeNetworkConnection(connectionId: string) {
+        return await this.apiClient.issueDeleteRequest(`/v1/network_connections/${connectionId}`);
+    }
+
+    /**
+     * Set routing policy for networkId 
+     * @param routingPolicy  the policy
```
Comment (Yinon-Hever) [inline src/fireblocks-sdk.ts:230]: redundant space between **routingPolicy** and **the**

File: src/fireblocks-sdk.ts (hunk 1 above)
Comment (yuval-fireblocks) [reply src/fireblocks-sdk.ts]: 👍

File: src/fireblocks-sdk.ts (hunk 2 above)
Comment (yuval-fireblocks) [reply src/fireblocks-sdk.ts]: 👍

File: src/types.ts [hunk 3]
```diff
@@ -391,37 +391,70 @@ export interface NetworkConnectionResponse {
...
+    crypto?: CustomCryptoRoutingDest | DefaultNetworkRoutingDest | NoneNetworkRoutingDest;
+    sen?: CustomFiatRoutingDest | DefaultNetworkRoutingDest | NoneNetworkRoutingDest;
+    signet?: CustomFiatRoutingDest | DefaultNetworkRoutingDest | NoneNetworkRoutingDest;
+    sen_test?: CustomFiatRoutingDest | DefaultNetworkRoutingDest | NoneNetworkRoutingDest;
+    signet_test?: CustomFiatRoutingDest | DefaultNetworkRoutingDest | NoneNetworkRoutingDest;
+}
+
+export interface NetworkIdRoutingPolicy {
+    crypto?: CustomCryptoRoutingDest | AutoNetworkRoutingDest | NoneNetworkRoutingDest;
+    sen?: CustomFiatRoutingDest | AutoNetworkRoutingDest | NoneNetworkRoutingDest;
```
Comment (Yinon-Hever) [inline src/types.ts:443]: @yuval-fireblocks what do u think about group all of them under one interface?

Comment (yuval-fireblocks) [reply src/types.ts:443]: I actually prefer keeping it like it is, for simplicity

**Pull request merged**